libs/map/src/components/legend/vl-map-legend.uig-css.ts

```diff
@@ -339,6 +353,9 @@ const mapLegendStyles: CSSResult = css`
 
     div.uig-map-legend--vertical div.uig-map-legend-item {
         line-height: normal;
+    div.uig-map-legend-icon-text {
```

Comment:
dit ziet er niet correct uit!

Reply:
Zoals besproken; dit komt door historiek herschrijving op `develop`-branch. Deze code komt nu ook niet meer in deze PR.